tests/popup-test.js

```diff
@@ -71,20 +74,10 @@ describe('addToDict(tempScript, htmlString, foundThis, key, category)', function
   });
 });
 
-describe('apps.json should be valid json', function () {
-
-  var fileIsOk;
-
-  beforeEach(() => {
-    sinon.stub(window, 'fetch');
-  });
-
-  afterEach(() => {
-    window.fetch.restore();
-  });
-
-
-  it('should return valid JSON', function () {
-    expect(1).to.equal(true);
+describe('vendors.json should be valid json', function () {
+  it('should say that the file exists', function () {
```

Comment:
`JSON.parse` should work:

```sh
> JSON.parse("[1]")
[ 1 ]
> JSON.parse("[1,]")
SyntaxError: Unexpected token ] in JSON at position 3
```

What I suggested last week was to use Node's `fs` package to read the JSON file into a string: https://docs.nodejitsu.com/articles/file-system/how-to-read-files-in-nodejs/